addons/docs/src/blocks/Props.tsx

```diff
@@ -63,7 +66,8 @@ const useArgs = (storyId: string, storyStore: StoryStore): [Args, (args: Args) =
   const updateArgs = useCallback((newArgs) => storyStore.updateStoryArgs(storyId, newArgs), [
     storyId,
   ]);
-  return [args, updateArgs];
+  const resetArgs = useCallback(() => storyStore.updateStoryArgs(storyId, initialArgs), [storyId]);
```

Comment:
This doesn't seem right:

```suggestion
  const resetArgs = useCallback((argNames?: string[]) => storyStore.resetStoryArgs(storyId, argNames), [storyId]);
```